src/utils/valueFixers.js

```diff
@@ -23,8 +24,25 @@ const toBoolean = function (value) {
   }
 };
 
-const stringToNumber = function (str) {
-  return Number(str);
+/** Given a number or a string representing a number, returns
+ *     a number. Needs both because it's used in setting valid
+ *     client values. Cash values will come in as numbers because
+ *     they have to be converted to monthly values. Other number
+ *     values, like age, will still be strings.
+ *
+ * @param {number|string} numberOrString
+ * @returns {number}
+ */
+const toNumber = function (numberOrString) {
+
+  let trueOrError = isNumberlike(numberOrString);
+
+  if (typeof trueOrError === `object`) {
```

Comment:
Might be clearer what we're expecting the possible return types to be if we use something like `trueOrError instanceof Error`--`object` is fairly vague.

Not a blocker; I'm fine with this going through as is.

Reply:
Oh! Didn't know `new TypeError("blah") instanceof Error` evaluated to `true`!

Reply:
@todo